pyuvdata/utils.py

```diff
@@ -1384,7 +1425,7 @@ def _reraise_context(fmt, *args):
     if len(args):
         cstr = fmt % args
     else:
-        cstr = six.text_type(fmt)
+        cstr = str(fmt)
 
     ex = sys.exc_info()[1]
```

Comment:
This entire function should be removed. It exists to handle exception chaining in python 2, which is automatic in python 3.

Reply:
I took it out and replaced it with the `raise from` pattern. @mkolopanis  or @steven-murray can you check what I did? I'm sorry, I probably should have done it in a PR to this branch but just pushed to this branch directly.

Reply:
This was an internal method, so I don't think taking it out without deprecation should be a problem, but I could be convinced otherwise.

Reply:
The changes look correct to me. A quick github search shows no direct calls to this function in `HERA-Team` which constitutes a large fraction of this package's user-base. Since python 3 has inherent exception chaining I think this is fine in the end. Will to change my opinion if others have stronger ones.